src/test/resources/Features/OCD-2966-newAdminNavAnnouncementsManagement.feature

```diff
@@ -0,0 +1,83 @@
+@Regression
+Feature: OCD-2966 New Admin nav - Announcements Management
+  This test checks whether all the functionalities work as expected for announcements
+
+Scenario Outline: User logs in as ROLE and verifies navigation is successful to the Announcements Management page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    Then I see "Announcements" as the page title for the Announcements management page
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify columns are displayed correct in the Announcements Management Page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    Then I see "Title Announcement Start Date End Date Public" columns in the Announcements page
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify error message is displayed when the title field is left blank
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    And I add an Announcement
+    And I set the "title" field to ""
+    Then I see "Title is required" error message 
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify error message is displayed when the start-date field is left blank
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    And I add an Announcement
+    And I set the "start-date" field to ""
+    Then I see "Start Date is required" error message 
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify error message is displayed when the end-date field is left blank
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    And I add an Announcement
+    And I set the "end-date" field to ""
+    Then I see "End Date is required" error message 
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify clicking cancel button in the Announcements section navigates back to the Announcement page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the Announcements Management page
+    And I add an Announcement
+    And I click cancel button in the Announcement section
+    Then I see "Announcements" as the page title for the Announcements management page
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: Verify adding announcement is saved and recorded on Announcements page and Announcements reports page
```

Comment:
Delete the announcement in the same scenario. That's what was done on one of the other recent AQA PRs. It's not super great because the scenario does a lot of things, but it works, and it also verifies that "delete" also works